Godeps/_workspace/src/k8s.io/kubernetes/pkg/util/io/writer.go

```diff
@@ -56,10 +58,11 @@ func (writer *NsenterWriter) WriteFile(filename string, data []byte, perm os.Fil
 		"--",
 	}
 
-	echo_args := append(base_args, "sh", "-c",
-		fmt.Sprintf("echo %q | cat > %s", data, filename))
+	echo_args := append(base_args, "sh", "-c", fmt.Sprintf("cat > %s", filename))
```

Comment:
I don't know all the things that call this. Is file name safe unquoted and unescaped?